### PR TITLE
fix(smallFixes): Small fixes for utils

### PR DESCRIPTION
**Pull request by PeepoFrog**
!(port < 0 || port > 65535)->port >= 0 && port <= 65535, RunCommand returns nill if shlex is executed with error, FileExist -> PathExist

### CODE REVIEW

**Review by MrLutik — APPROVED** [review]
All good. ++ 
@andrsj waiting for your approval

**Comment by andrsj** [comment]
All good

**Review by andrsj — APPROVED** [review]
All good

**Pull request merged**
